prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/humanname_to_xpn_fn_mapping_hl7_dest.hl7

```diff
@@ -0,0 +1,5 @@
+MSH|^~\&#|STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|Simple Report^CDPH000085^CLIA|CDPH CA REDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO|CDPH_CID^2.16.840.1.114222.4.1.214104^ISO|20230213111654-0500||ORU^R01^ORU_R01|32361_U47 Mega Case Take 1_3015894676_04608646_11034|T|2.5.1|||NE|NE|USA|UNICODE UTF-8|||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
```

Comment:
The test file has a lot of elements outside of the ones being mapped. Most of these should be cleared out as future mapping work may break these tests.

Reply:
I am aware of those scope creeping and I mainly did it as exercise, thanks for point it out